NEWS.md

```diff
@@ -210,6 +210,21 @@
   reflect that it is a lossy operation that flattens all nested
   quosures.
 
+* The particle "lang" has been renamed to "call":
+
+    - `lang()` has been renamed to `call2()`.
+    - `new_language()` has ben renamed to `new_call()`.
+    - `is_lang()`, `is_unary_lang()`, and `is_binary_lang()` have been
```

Comment:
Should we take this opportunity to eliminate `is_unary_call` and `is_binary_call` in favour of the `n` argument to `is_call`?

Reply:
Sounds good.